util/tests/test-stream.js

```diff
@@ -171,6 +171,20 @@ o.spec("stream", function() {
 
 			o(b()).equals(undefined)
 		})
+		o("combine will throw with a helpful error if given non-stream values", function () {
+			var spy = o.spy()
+			var a = Stream(1)
+			var thrown = null;
+			try {
+				var b = Stream.combine(spy, [a, ''])
+			} catch (e) {
+				thrown = e
+			}
+
+			o(thrown).notEquals(null)
+			o(thrown.constructor === TypeError).equals(false)
```

Comment:
I don't disagree - I was trying to find a clean-ish way to determine that it was the "right" error (i.e. an error that was intentionally thrown, not `cannot access property 'changed' of undefined (stream._state)`.